common/changes/@rushstack/ts-command-line/ts-command-line-recommendation-updates_2024-05-15-20-16.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/ts-command-line",
+      "comment": "Mark `onDefineParameters` as deprecated and update README accordingly because defining parameters causes issues when the comiler targets >=es2022.",
```

Comment:
```suggestion
      "comment": "Mark `onDefineParameters` and `onDefineUnscopedParameters` as deprecated and update README accordingly because defining parameters causes issues when the compiler targets >=es2022.",
```